source/administration/plugins.rst

```diff
@@ -12,12 +12,68 @@ Consider using a plugin in the following scenarios:
  - When you want to extend Mattermost functionality to meet a specific, complex requirement
  - When you want to build integrations that are managed by your Mattermost server
 
-Plugins are fully supported by both Team and Enterprise Editions. Plugins may have one or both of the following parts:
+Plugins are fully supported by both Team and Enterprise Editions. 
 
- - **Webapp plugins (Beta)**: Customize the Mattermost user interface by adding to the channel header, overriding the ``RHS``, or even rendering a custom post type. All this is possible without having to fork the source code and rebase on every Mattermost release. For a sample plugin, see `our Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__.
- - **Server plugins (Release Candidate)**: Run a Go process alongside the server, filtering messages or integrating with third-party systems such as Jira, GitLab or Jenkins. For a sample plugin, see `our Jira plugin <https://github.com/mattermost/mattermost-plugin-jira>`__.
 
-Browse available plugins at `https://integrations.mattermost.com/ <https://integrations.mattermost.com/>`__.
+Plugin Marketplace
+-------
+The Plugin Marketplace is a collection of plugins that can greatly increase the value and capabilities of your Mattermost deployment.  End-users can see increased productivity through quick access to systems such as Jira, Zoom, Webex, and GitHub.  Mattermost Administrators can discover new plugins and quickly deploy them to their servers, including High availability clusters.  It is available in 5.16 and is accessed via **Main Menu > Plugin Marketplace**.  Our Marketplace listings will continue to expand as we add new features and plugins that our customers request.  
+
+.. image:: https://user-images.githubusercontent.com/915956/66891467-1b18eb80-ef9e-11e9-9de3-37a3c5899bd8.png
+
+Installing a Plugin
+~~~~~~~~~~~~~~~~
+
+When a new plugin becomes available on the marketplace, a new listing with a button labeled **Install** will be displayed.  After clicking **Install**, the latest plugin binary is downloaded from it's respective github repository and installed on the server.  If there is a cluster present, the plugin will be automatically distributed to each server automatically.   
```

Comment:
Do we really need to mention clusters here? It seams like an over complication.

Reply:
Removed it